src/resolver.rs

```diff
@@ -81,6 +81,8 @@ abi_resolver! {
         9, "self_id" => self_id::SelfId,
         10, "gas" => gas::Gas,
         11, "block_height" => block_height::BlockHeight,
-        100, "poseidon_hash" => poseidon_hash::PoseidonHash
+
+        100, "poseidon_hash" => poseidon_hash::PoseidonHash,
+        101, "verify_proof" => verify_proof::ProofVerification
```

Comment:
As mentioned the convention should reflect the host call name so this should be `verify_proof::VerifyProof`